openlibrary/templates/books/custom_carousel.html

```diff
@@ -1,5 +1,10 @@
 $def with(books=[], title="", url="", key="", min_books=1, load_more=None, test=False)
 
+$ ctx.setdefault("links", [])
+$ slick_font = '<link rel="preload" href="/static/css/fonts/slick.woff" as="font" type="font/woff">'
```

Comment:
That slick font still seems pretty gratuitous to me since all it does is provide 2 icons. Personally I would prefer that we drop it altogether. We could use an SVG as an alternative.

Have you considered using SVG/CSS and dropping this CSS as an alternative? :
```

/* Icons */
@font-face
{
    font-family: 'slick';
    font-weight: normal;
    font-style: normal;

    src: url(/static/css/fonts/slick.eot);
    src: url(/static/css/fonts/slick.eot?#iefix) format('embedded-opentype'),
        url(/static/css/fonts/slick.woff) format('woff'),
        url(/static/css/fonts/slick.ttf) format('truetype'),
        url(/static/css/fonts/slick.svg#slick) format('svg');
}
```

Reply:
Opened as a new issue: #3239 -- I'm in support of this approach, this PR seems like a reasonable intermediary step.